charts/seed-controlplane/charts/kube-apiserver/templates/kube-apiserver.yaml

```diff
@@ -53,7 +53,11 @@ spec:
         imagePullPolicy: IfNotPresent
         command:
         - /hyperkube
+        {{- if semverCompare "< 1.15" .Values.kubernetesVersion }}
```

Comment:
Yes, we could do that, however, this would result in a needless restart of all hyperkube components for older clusters. As we don’t gain much out of it (imho, the if-else-statement is not complex) I would not do it now.